package.json

```diff
@@ -1,24 +1,29 @@
 {
-  "name": "yt-to-anchor",
+  "name": "youtube-to-anchorfm",
   "version": "1.0.0",
```

Comment:
Should we bump the version to 1.1 or even 2?

Reply:
Yeah, we should bump the version